File: thirdday/prime numbers/code.cpp
# wrong code
bool is_prime(int n)
{
    for (int i = 2; i < n; i++)
        if (n % i == 0)
            return false;
    return true;
}

# correct code
bool is_prime(int n)
{
    if(n==1 || n==0)
        return false;
    
    for (int i = 2; i < n; i++)
        if (n % i == 0)
            return false;
    return true;
}

File: thirdday/prime numbers/code.py
# Wrong code
def is_prime(n):
    for i in range(2, n):
        if n % i == 0:
            return False
    return True

# correct code
def is_prime(n):
    if(n==0 OR n==1):
        return False
    for i in range(2, n):
        if n % i == 0:
            return False
    return True
